src/recordings.js

```diff
@@ -64,7 +64,6 @@ export function retrieveListOfSpeakers() {
 
       // play the audio associated with that specific index
       let audio = new Audio(audioURL)
-      audio.preload = 'none'
```

Comment:
I think this was here as a result of cargo cult programming, haha! On the search result page, this prevents the recordings of ALL of the results loading all at once.